src/main/java/com/sap/oss/phosphor/fosstars/data/artifact/experimental/owasp/data/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * The package contains data providers which fetch data from OWASP Dependency-Check.
```

Comment:
This package doesn't contain data providers. Did you use a particular JSON schema to generate them? If so, please mention that. It would be also good to describe how exactly you generated the original classes.

Reply:
@artem-smotrakov is this ok? I generated exactly like that.
```suggestion
 * The package contains POJOs which was manually generated based on a sample OWASP Dependency-Check
 * result.
```

Reply:
Ok.